docs/pages/api-reference/liveblocks-react.mdx

```diff
@@ -10,21 +10,1134 @@ alwaysShowAllNavigationLevels: false
+          return __getUserIds__(matchingUsers);
+        }
+      }}
+
+      // Other props
+      // ...
+    >
+      {/* children */}
+    </LiveblocksProvider>
+  );
+}
+```
+
+#### LiveblocksProvider for Node.js [#LiveblocksProviderNode]
+
+To use `@liveblocks/client` in Node.js, you need to provide [`WebSocket`][] and
+[`fetch`][] polyfills. As polyfills, we recommend installing [`ws`][] and
+[`node-fetch`][].
+
+```bash
+npm install ws node-fetch
+```
+
+Then, pass them to the `LiveblocksProvider` polyfill option as below.
+
+```tsx
+import { LiveblocksProvider } from "@liveblocks/react/suspense";
+import fetch from "node-fetch";
+import WebSocket from "ws";
+
+function App() {
+  return (
+    <LiveblocksProvider
+      polyfills={{
+        fetch,
+        WebSocket,
+      }}
+
+      // Other props
+      // ...
+    >
+      {/* children */}
+    </LiveblocksProvider>
+  );
+}
+```
+
+Note that `node-fetch` v3+
+[does not support CommonJS](https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md#converted-to-es-module).
+If you are using CommonJS, downgrade `node-fetch` to v2.
+
+#### LiveblocksProvider for React Native [#LiveblocksProviderReactNative]
+
+To use `@liveblocks/client` with [React Native](https://reactnative.dev/), you
+need to add an [`atob`][] polyfill. As a polyfill, we recommend installing
+[`base-64`][].
+
+```bash
+npm install base-64
+```
+
+Then you can pass the `decode` function to our `atob` polyfill option when you
+create the client.
+
+```ts
+import { LiveblocksProvider } from "@liveblocks/react/suspense";
+import { decode } from "base-64";
+
+function App() {
+  return (
+    <LiveblocksProvider
+      polyfills={{
+        atob: decode,
+      }}
+
+      // Other props
+      // ...
+    >
+      {/* children */}
+    </LiveblocksProvider>
+  );
+}
+
+```
+
+### createLiveblocksContext
+
+<Banner>
+
+This used to be the default way to start your app, but now it’s recommend for
+advanced usage only. We generally recommend using ... TODO ... unless you need
+to define multiple room types in your application.
+
+</Banner>
+
+Creates a [`LiveblocksProvider`][] and a set of typed hooks. We recommend using
+it in `liveblocks.config.ts` and re-exporting your typed hooks like below.
+
+While [`createRoomContext`](#createRoomContext) offers APIs for interacting with
+rooms (e.g. Presence, Storage, and Comments),
+[`createLiveblocksContext`](#createLiveblocksContext) offers APIs for
+interacting with Liveblocks features that are not tied to a specific room (e.g.
+Notifications).
+
+```tsx file="liveblocks.config.ts"
+import { createClient } from "@liveblocks/client";
+import { createRoomContext, createLiveblocksContext } from "@liveblocks/react";
+
+const client = createClient({
+  // publicApiKey: "",
+  // authEndpoint: "/api/liveblocks-auth",
+  // throttle: 100,
+});
+
+// ...
+
+export const { RoomProvider } = createRoomContext(client);
+
+export const {
+  LiveblocksProvider,
```

Comment:
Not sure where we should mention it, but _this_ `LiveblocksProvider`, in contrast to a "normal" imported `LiveblocksProvider`, does not take any props. Those are already bound from the `client`.